program-runtime/src/log_collector.rs

```diff
@@ -69,15 +69,23 @@ impl From<LogCollector> for Vec<String> {
 #[macro_export]
 macro_rules! ic_logger_msg {
     ($log_collector:expr, $message:expr) => {
-        $crate::log_collector::log::debug!("{}", $message);
+        $crate::log_collector::log::debug!(
+            target: "solana_runtime::message_processor::stable_log",
+            "{}",
+            $message
+        );
         if let Some(log_collector) = $log_collector.as_ref() {
             if let Ok(mut log_collector) = log_collector.try_borrow_mut() {
                 log_collector.log($message);
             }
         }
     };
     ($log_collector:expr, $fmt:expr, $($arg:tt)*) => {
-        $crate::log_collector::log::debug!($fmt, $($arg)*);
+    $crate::log_collector::log::debug!(
```

Comment:
fix indent?

Reply:
Stupid `cargo fmt` gets lost in macro definitions